docs/scarpet/api/Auxiliary.md

```diff
@@ -268,6 +268,14 @@ Example usages:
   // not a problem in apps
 </pre>
 
+### `item_display_name(item)`
```

Comment:
new functions need to be added to the supported editor keywords

Reply:
i m sorry that i dont know how editor file works... 
and npp is a illegal software in my country. i dont know if i allowed to do so.

Reply:
so i added the idea`s one now.